Maven Build not working for AIAgent

Going by documentation the maven dependency is not working. Does not compile.
Have tried including jvm dependency.  None of the ai.koog libraries are available.

        <dependency>
            <groupId>ai.koog</groupId>
            <artifactId>koog-agents</artifactId>
            <version>0.3.0</version>
        </dependency>


    fun main() {
        runBlocking {
            val apiKey = System.getenv("OPENAI_API_KEY") // or Anthropic, Google, OpenRouter, etc.

            val agent = AIAgent(
                executor = simpleOpenAIExecutor(apiKey), // or Anthropic, Google, OpenRouter, etc.
                systemPrompt = "You are a helpful assistant. Answer user questions concisely.",
                llmModel = OpenAIModels.Chat.GPT4o
            )

            val result = agent.run("Hello! How can you help me?")
            println(result)
        }
    }

We are also facing the same issue.

when we added dependency 
`<dependency>
    <groupId>ai.koog</groupId>
    <artifactId>koog-agents</artifactId>
    <version>0.3.0</version>
</dependency>`
We are getting below error:
`
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.jetbrains.kotlin:kotlin-maven-plugin:2.2.0:compile (compile-kotlin) on project LLM-bot: Compilation failure: Compilation failure:
[ERROR] Service.kt:3:8 Unresolved reference 'ai'.
[ERROR] Service.kt:4:8 Unresolved reference 'ai'.
[ERROR] Service.kt:5:8 Unresolved reference 'ai'.
[ERROR] Service.kt:6:16 Unresolved reference 'coroutines'.
[ERROR] Service.kt:15:16 Unresolved reference 'AIAgent'.
[ERROR] Service.kt:20:17 Unresolved reference 'AIAgent'.
[ERROR] Service.kt:21:36 Unresolved reference 'simpleOpenAIExecutor'.
[ERROR] Service.kt:24:36 Unresolved reference 'OpenAIModels'.
[ERROR] Service.kt:28:57 Unresolved reference 'runBlocking'.
[ERROR] Service.kt:29:29 Cannot infer type for this parameter. Specify it explicitly.
[ERROR] Service.kt:29:29 Unresolved reference. None of the following candidates is applicable because of a receiver type mismatch:
[ERROR] fun <T, R> T.run(block: T.() -> R): R
[ERROR]     [R|Contract description]
[ERROR]      <
[ERROR]         CallsInPlace(block, EXACTLY_ONCE)
[ERROR]     >
[ERROR] -> [Help 1]
`
and if we change dependency to 
`		<dependency>
			<groupId>ai.koog</groupId>
			<artifactId>koog-agents-jvm</artifactId>
			<version>0.3.0</version>
		</dependency>
`
we are getting :
`
Caused by: java.lang.NoClassDefFoundError: kotlin/uuid/Uuid
        at ai.koog.agents.core.agent.AIAgentKt.AIAgent$default(AIAgent.kt:479)
        at org.finos.springbot.tool.llm.ChatGPTLLMService.afterPropertiesSet(ChatGPTLLMService.kt:20)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1865)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1814)
        ... 42 more
Caused by: java.lang.ClassNotFoundException: kotlin.uuid.Uuid
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
`
Please let us know, do we need to add any other dependency.

--- a/README.md
+++ b/README.md
@@ -98,7 +98,7 @@ Please check the [libs.versions.toml](gradle/libs.versions.toml) to know more ab
     ```
     <dependency>
         <groupId>ai.koog</groupId>
-        <artifactId>koog-agents</artifactId>
+        <artifactId>koog-agents-jvm</artifactId>
         <version>0.3.0</version>
     </dependency>
     ```

--- a/docs/docs/index.md
+++ b/docs/docs/index.md
@@ -74,7 +74,7 @@ To use Koog, you need to include all necessary dependencies in your build config
     ```
     <dependency>
         <groupId>ai.koog</groupId>
-        <artifactId>koog-agents</artifactId>
+        <artifactId>koog-agents-jvm</artifactId>
         <version>LATEST_VERSION</version>
     </dependency>
     ```

